client/src/App.css

```diff
@@ -0,0 +1,6 @@
+#root{
```

Comment:
Don't add styles for the root div !!!
It will affect all of our work. Only add styles for your specific component and/or general styles that we'll all need.

Reply:
I have to style the root div because I want to give it a min-height: 100vh. If I did not do so, the footer will be in the middle of the page. See?
![Screenshot from 2019-05-29 07-37-03](https://user-images.githubusercontent.com/35868173/58529707-9a22a880-81e4-11e9-8891-855542a62617.png)
And even giving the div height of 100vh won't change anything if I did not use flex. 
This won't affect your work at all in case I removed this line of code:
`justify-content: space-between`

Do you have a better idea? @ahmedisam99